cellpose/gui/io.py

```diff
@@ -707,3 +711,41 @@ def _save_sets(parent):
     del dat
     #print(parent.point_sets)
     print("GUI_INFO: %d ROIs saved to %s" % (parent.ncells, base + "_seg.npy"))
+
+def process_tiff_image(tiff_file):
+    """
+    Processes a multi-layer TIFF image located at `tiff_file`.
+    Converts each layer to opacity format and returns a list of processed images.
+
+    Parameters:
+    - tiff_file (str): Path to the multi-layer TIFF image.
+
+    Returns:
+    - processed_images (list): List of processed PIL Image objects.
+    """
+    processed_images = []
+    with Image.open(tiff_file) as img:
+        for frame in ImageSequence.Iterator(img):
+            processed_frame = convert_grayscale_to_opacity(frame)
+            processed_images.append(processed_frame)
+    return processed_images
+
+def convert_grayscale_to_opacity(frame):
+    """
+    Converts a grayscale image frame to opacity format.
+    Converts the image to 16-bit grayscale, extracts the alpha channel,
+    and merges it with a white background to create an opacity-enhanced image.
+
+    Parameters:
+    - frame (PIL.Image): Input grayscale image frame.
+
+    Returns:
+    - final_image (PIL.Image): Processed opacity-enhanced image.
+    """
+    frame = frame.convert("I;16B")
+    image_np = np.array(frame)
+    alpha_np = (image_np >> 8).astype(np.uint8)
```

Comment:
Konnte leider nicht nachschauen aber gibt es einen besseren weg es direkt in 8bit zu konvertieren?